mobify/platform-scaffold
# Update link to Quick Start guide

The old URL links a page that doesn't exist anymore. 

## How to test-drive this PR
- Please verify that https://docs.mobify.com/progressive-web/latest/getting-started/quick-start/ is the correct URL for the Quick Start guide
